info/util.go

```diff
@@ -0,0 +1,116 @@
+package info
+
+import (
+	"crypto/x509/pkix"
+	"fmt"
+	"strings"
+)
+
+// func certPublic(cert *x509.Certificate) string {
```

Comment:
It's imported from somewhere else, and most of it is not needed; I wasn't sure what all of it I would need at first. I've removed all the commented out parts.